docs/UserGuide.md

```diff
@@ -190,3 +190,66 @@ Action | Format, Examples
 **Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
 **List** | `list`
 **Help** | `help`
+
+### Editing an event : `eedit`
+
+Edits an existing event in the event list.
+
+Format: `eedit INDEX [n/NAME] [at/START_TIME] [end/END_TIME] [d/DESCRIPTION] [a/ADDRESS] [z/ZOOM] [dt/TAG_DELETED]…​ [t/TAG_ADDED]…​`
+
+Take note of the following time format:
+Start time should be of format “dd-MM-yyyy HH:mm” (date-month-year Hour:minutes in 24 hr format).
```

Comment:
This can be wrapped within a alert-info div:
``` suggestion
<div markdown="span" class="alert alert-info">
:information_source: **Note:** Start time should be of format “dd-MM-yyyy HH:mm” (date-month-year Hour:minutes in 24 hr format).
</div>
```